network-api/networkapi/mozfest/models.py

```diff
@@ -275,9 +240,10 @@ class MozfestHomepage(MozfestPrimaryPage):
         TranslatableField("cta_button_label"),
         SynchronizedField("cta_button_destination"),
         TranslatableField("banner_heading"),
-        TranslatableField("banner_cta_label"),
-        TranslatableField("banner_carousel"),
-        SynchronizedField("banner_video"),
+        SynchronizedField("banner"),
+        TranslatableField("banner_meta"),
+        TranslatableField("banner_text"),
+        TranslatableField("banner_link_text"),
```

Comment:
I think we should add the banner_link_url as a synchronized field. If the translatable_fields is explicitly defined, we need to list all fields. Fields that are not listed are only copied on creation of the translated model and are not updated thereafter, which would mean they can get out of sync with the source page.

Reply:
:+1: 025abec2e96688482e1420869fa7f227aefb9a0c